Source/Particles/Deposition/CurrentDeposition.H

```diff
@@ -68,6 +69,7 @@ void doDepositionShapeN(const GetParticlePosition& GetPosition,
                         amrex::Real* cost,
                         const long load_balance_costs_update_algo)
 {
+    amrex::ignore_unused(dt);
```

Comment:
Could we simply remove this variable from the list of arguments for this function?

Reply:
Thank you, you're right. Done in https://github.com/ECP-WarpX/WarpX/pull/2937/commits/9a7e85f95650e7c2365e673b85c850e9dde06c6a.